lisp/ess-custom.el

```diff
@@ -887,7 +887,7 @@ See `ess-style-alist' for for an overview of ESS indentation."
      (ess-indent-function-declaration . t)
      (ess-indent-from-lhs . t)
      (ess-indent-with-fancy-comments . t))
-    
```

Comment:
this mode looks very nice. Especially the fact that it only cleans whitespace if the buffer was already clean when it was opened.

I reverted the whitespace and added a bullet to the news file.